frontend-react/e2e/helpers/utils.ts

```diff
@@ -139,3 +139,18 @@ export function toDateWithTime(date: string, time?: string) {
     }
     return toDateTime;
 }
+
+export function removeTimestamp(filename: string) {
```

Comment:
maybe rename function to `removeDateTime`

Reply:
i'll rename it. i wanted to keep the extension (in this case `.hl7`) intact in case things change + keep the specificity of filetype.